res/static/fragment_callback.js

```diff
@@ -0,0 +1,14 @@
+document.addEventListener('DOMContentLoaded', function() {
```

Comment:
Good idea, but I think it should also work to just move the script tag below the form. Will add a separate commit for this that also tackles `forward.js`.